api/routers/report.py

```diff
@@ -102,37 +111,107 @@ async def update_reports(old_user_id: int, new_user_id: int, token: str):
 
 
 @router.post("/v1/report", tags=["Report"])
-async def insert_report(token: str, detections: List[detection]):
+async def insert_report(request : Request,
+    detections: List[detections],
+    manual_detect: int = Query(0, ge=0, le=1),
+    ):
     '''
-        Work in progress 
-        Insert report.
+        Inserts detections to the plugin database.
     '''
-    await verify_token(token, verification='verify_ban', route='[POST]/v1/report/')
+    client_host = request.client.host
```

Comment:
No

Reply:
Need more than 'No'

Reply:
![image](https://user-images.githubusercontent.com/40169115/148830945-00a90a83-8c5e-4c1c-93dc-d232e5018a92.png)

Reply:
This is not a "no" still, as declared by the warning - when installing or using the Plugin your IP is used to send data to the server. This does not break the accepted terms.

Reply:
If the warning said that the IP would not be sent, which would be impossible, then this would not be a feasible change - though it's part of the accepted terms and can be used to blacklist & secure our own systems against threat.

Reply:
I don't think it is reasonable to log IP as part of the debugging

Reply:
Okay that's an understandable and different concern - for security purposes would we want to handle it in a different way if a bad route is hit multiple times in succession?

Reply:
the nginx ratelimter will handle that